### PR TITLE
add `Paul Miller`

### PR DESCRIPTION
**Pull request by paulmillr**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -17,8 +17,7 @@ Amas is a [recursive acronym](https://en.wikipedia.org/wiki/Recursive_acronym).
 - [Wes Bos](https://github.com/wesbos/ama) - Full stack developer, independent author, speaker, teacher and JavaScripter.
 - [Anselm Hannemann](https://github.com/anselmh/AMA) - Freelance front-end developer, author of WDRL, co-organizer of Nightlybuild.io.
 - [Blain Smith](https://github.com/blainsmith/ama) - Partner/Engineer at MadGlory and Odd Networks. Rebel Geek.
-
--
```
Comment (arthurvr) [inline readme.md:21]: don't remove these :)

File: readme.md [hunk 2]
```diff
@@ -17,8 +17,9 @@ Amas is a [recursive acronym](https://en.wikipedia.org/wiki/Recursive_acronym).
 - [Wes Bos](https://github.com/wesbos/ama) - Full stack developer, independent author, speaker, teacher and JavaScripter.
 - [Anselm Hannemann](https://github.com/anselmh/AMA) - Freelance front-end developer, author of WDRL, co-organizer of Nightlybuild.io.
 - [Blain Smith](https://github.com/blainsmith/ama) - Partner/Engineer at MadGlory and Odd Networks. Rebel Geek.
-
+- [Paul Miller](https://github.com/paulmillr/ama) — Creator of Brunch.io, Chaplin, es6-shim and many NPM modules. Entrepreneur & frequent traveller.
```
Comment (arthurvr) [inline readme.md:20]: > NPM

Never capitalize [npm](https://docs.npmjs.com/misc/faq#is-it-npm-or-npm-or-npm)

**Comment by paulmillr** [comment]
/ (╯°□°）╯︵ ┻━┻

**Comment by sindresorhus** [comment]
Fixed it for ya ;)

**Comment by sindresorhus** [comment]
Hey. Want to ask you a favor. Would be cool if you could ask some questions on the other AMAs in the list (not my AMA). I think AMAs are a great opportunity to learn from each other. Asking a question is also a good way to encourage receiving one ;)

For inspiration of what to ask, you could take a look at the questions in some other AMAs, like:
https://github.com/passy/ama/issues?q=
https://github.com/sebmck/ama/issues?q=
https://github.com/wesbos/ama/issues?q=
https://github.com/sindresorhus/ama/issues?q=

**Pull request closed without merging**